docs/develop/api-reference/response-objects.md

```diff
@@ -0,0 +1,53 @@
+---
+sidebar_position: 5
+sidebar_label: Response objects
+title: Response object API reference
+description: Learn how to interact with the response object in the Momento API.
+slug: /develop/api-reference/response-objects
+---
+
+# Response objects
+
+These are the baseline responses for all commands. Some commands will add more data and functionality.
+
+Commands fall *generally* into two categories. Those which respond with Success or Error and those which respond with Hit, Miss, or Error.
+
+### Response
+
+A base class for all responses.
+
+#### Constructor
+
+- By default, no parameters.
+- If any values are needed from the GRPC response, pass in the whole object. This makes it easier to use more values later, and it moves code out of the often quite bloated client class.
+
+#### Methods
+
+- toString(): string - a brief description of the response for debugging purposes. Each response should override toString() as needed. The default is to display the type of response (Hit, Miss, Success, etc.)
+
+### Error extends Response
```

Comment:
Not sure if the `extends Response` part adds value here.  That's pretty specific to the individual programming languages.